{{cookiecutter.project_slug}}/clients/web/react/src/utils/get-cookie.ts

```diff
@@ -6,12 +6,12 @@
  * Usage:
```

Comment:
Love these changes. Especially the named export over default!

Reply:
But beware :

https://github.com/thinknimble/tn-spa-bootstrapper/blob/4c5cc6d638cb4a433013293600204923c865154a/%7B%7Bcookiecutter.project_slug%7D%7D/clients/web/react/src/services/apollo-client.ts#L4

^ let's update this to be named import!